src/components/Workout/ButtonSave.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Button from '../common/Button'
+
+export default function ButtonSave({ formState, form }) {
+  return (
+    <>
```

Comment:
<> und </> brauchst du nicht da du ja schon mit <Button> eine in sich geschlossene Komponente hast.